lib/ex_pesa/Mpesa/stk.ex

```diff
@@ -77,4 +77,63 @@ defmodule ExPesa.Mpesa.Stk do
   def request(_) do
     {:error, "Required Parameters missing, 'phone, 'amount', 'reference', 'description'"}
   end
+
+  @doc """
+  STK PUSH Transaction Validation
+
+  ## Configuration
+
+  Add below config to dev.exs / prod.exs files (at this stage after STK, the config keys should be there)
```

Comment:
Should we have this here? I think having this in the setup docs is better.

Reply:
@kamalogudah yeah it should be there, we wills till have it at the cofig level too. For every api, there are different configs and these we add them at the point we need them, like say B2B configs, we only add them when,  there is need to consume B2B, so the above are just specifi for STK validation